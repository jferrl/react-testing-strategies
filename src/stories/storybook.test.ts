import testStorySnapshots from '@storybook/addon-storyshots';

testStorySnapshots({
    framework: 'react'
});
