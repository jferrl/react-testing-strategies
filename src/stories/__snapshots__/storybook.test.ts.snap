// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/NavBar Default 1`] = `
<nav
  className="navbar navbar-expand navbar-dark bg-primary"
  data-testid="mars-rover-photos-navbar"
>
  <a
    className="navbar-brand"
    data-testid="mars-rover-photos-navbar-brand"
    href="#"
  >
    Mars Rover Photos
  </a>
</nav>
`;
